pipeline.py

```diff
@@ -72,26 +85,60 @@ class TextProcessingPipeline:
     """
 
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        articles = self.corpus_manager.get_articles().values()
+        tokens = []
+        for article in articles:
+            article_raw_text = article.get_raw_text()
+
+            for token in self._process(article_raw_text):
+                tokens.append(token.get_cleaned())
+
+            article.save_as(' '.join(tokens), ArtifactType.cleaned)
 
     def _process(self, raw_text: str):
         """
         Processes each token and creates MorphToken class instance
         """
-        pass
+
+        text = re.sub(r"[^а-я\s]", "", raw_text)
+        tokens = text.split()
```

Comment:
as I told you: when there is a way to avoid regex - use it

Reply:
changed